.circleci/config.yml

```diff
@@ -87,128 +65,88 @@ _snyk_options: &snyk_options
   fail-on-issues: true
 
 jobs:
-  build:
+  set_app_version:
     executor: builder
     steps:
-      - checkout
-      - set_app_version
-      - attach_workspace:
-          at: ~/app
-      - restore_cache:
-          keys:
-            - dependencies-cache-{{ checksum "package.json" }}
-            - dependencies-cache
       - run:
-          name: Install dependencies
-          command: npm ci
-      - run:
-          name: Build css
-          command: npm run css-build
-      - run:
-          name: Minify js and css
-          command: npm run minify-js-css
-      - save_cache:
-          key: dependencies-cache-{{ checksum "package.json" }}
-          paths:
-            - ~/.cache
-      - persist_to_workspace:
-          root: ~/app
-          paths: .
+          name: create app version string
+          command: |
+            DATE=$(date '+%Y-%m-%d')
+            VERSION=${DATE}.${CIRCLE_BUILD_NUM}
+            echo "Created version string: ${VERSION}"
+            echo "export VERSION=$VERSION" >> $BASH_ENV
+      - mem/remember:
+          env_var: APP_VERSION
+          value: "${VERSION}"
 
-  lint:
+  test_lint_unit_pact:
     executor: builder
     steps:
       - checkout
-      - attach_workspace:
-          at: ~/app
-      - restore_cache:
-          key: dependencies-cache-{{ checksum "package.json" }}
+      - setup_remote_docker:
+          docker_layer_caching: true
       - run:
-          name: Lint
-          halt_build_on_fail: false
-          command: npm run lint
-
-  test:
-    executor: builder
-    steps:
-      - checkout
-      - attach_workspace:
-          at: ~/app
-      - restore_cache:
-          key: dependencies-cache-{{ checksum "package.json" }}
+          name: Build Development Image
+          command: docker build --target=ci -t prepare-a-case-app -f Dockerfile .
       - run:
-          name: Unit tests
-          command: npm run unit-test
-      - store_test_results:
-          path: test-results
-      - store_artifacts:
-          path: coverage
+          name: Lint Test
+          command: docker run -it prepare-a-case-app /usr/src/app/node_modules/.bin/standard
+      - run:
+          name: Unit Test
+          command: docker run -e ENABLE_HEARING_OUTCOMES=false -it prepare-a-case-app /usr/src/app/node_modules/.bin/jest --config=./tests/jest.config.js --detectOpenHandles --ci --forceExit
```

Comment:
is this correct? we've now got a dependency on a feature flag within the CI pipeline

Reply:
Always was, if that feature flag isn't false the unit tests fail, so it's forced in CI.